cirrus/server/tests/conftest.py

```diff
@@ -51,3 +52,274 @@ def fml():
 @fixture
 def exception():
     return Exception("some error")
+
+
+@fixture(name="reset_glean", scope="function", autouse=True)
+def fixture_reset_glean():
+    testing.reset_glean(application_id="cirrus-test", application_version="0.1.0")
+
+
+@fixture
+def get_telemetry():
+    return initialize_glean()
+
+
+@fixture
+def recipes():
+    return {
+        "data": [
+            {
+                "slug": "cirrus-test-1",
```

Comment:
It might be good add mozilla-nimbus-shared and run these through the schema validator?